src/main/java/seedu/address/model/person/Age.java

```diff
@@ -0,0 +1,62 @@
+package seedu.address.model.person;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.AppUtil.checkArgument;
+
+/**
+ * Represents a Person's age in ORGANice.
+ * Guarantees: immutable; is valid as declared in {@link #isValidAge(String)}
+ */
+public class Age {
+
+
+    public static final String MESSAGE_CONSTRAINTS =
+            "Age should only contain numbers, and it must be more than 0";
```

Comment:
say less than 150 too?

Reply:
Rectified in commit "Rectify problems pointed out in PR Review"